forge-gui/res/cardsfolder/upcoming/astarions_thirst.txt

```diff
@@ -2,6 +2,6 @@ Name:Astarion's Thirst
 ManaCost:3 B
 Types:Instant
 A:SP$ ChangeZone | ValidTgts$ Creature | TgtPrompt$ Select target creature | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBPutCounter | StackDescription$ Exile {c:Targeted}. | SpellDescription$ StackDescription
```

Comment:
I don't believe this will look right in the card detail pane.

Reply:
Looking at the card detail pane and the stack description, I think I've come to understand that the `{c:Targeted}` tag is causing some problem when used in the `SpellDescription$` string. I've revised the spell description to state the full spell text as written in oracle, but kept in the `StackDescription$` string in line 5. Admittedly there is probably a more efficient way to do this, but it seems that this combination is now successfully presenting the correct stack description and card detail text in testing.

Reply:
The tokens (like `{c: whatever}`) are only for StackDescription... they aren't parsed for SpellDescription

Reply:
In terms of efficiency it'd have been easier to leave line 5 alone and just set line 4 `SpellDescription$` to the first sentence of the oracle text

But this works!